api/controllers/countries.js

```diff
@@ -108,6 +108,17 @@ function groupResults (results) {
       count: count
     });
   });
-  final = sortBy(final, ['name']);
   return final;
 }
+
+/**
+* Order results after grouping them
+*
+* @param {array|string} results - The grouped results
+* @param {object} query - Query object from Hapi
+* @return {array} results - Ordered results
+*/
+function orderResults (results, query) {
```

Comment:
If this is going to be the same for each controller, is there anyway to just put it in one central place?